airflow/providers/amazon/aws/example_dags/example_s3_to_redshift.py

```diff
@@ -35,11 +35,13 @@
 # [END howto_operator_s3_to_redshift_env_variables]
 
 
+@task(task_id='setup__add_sample_data_to_s3')
 def _add_sample_data_to_s3():
```

Comment:
I'm wondering if we should skip the `_`?

Reply:
I'm fine with this. I wanted to change as little as possible but I was wavering on this as well. Will update.